src/Integration/Exclusions/AnalyzableFileIndicator.cs

```diff
@@ -87,6 +87,10 @@ public bool ShouldAnalyze(string filePath)
             exclusions != null &&
             exclusions.Any(x => IsMatch(x, filePath));
 
-        private bool IsMatch(string pattern, string filePath) => globPatternMatcher.IsMatch(pattern, filePath);
+        private bool IsMatch(string pattern, string filePath)
+        {
+            filePath = filePath.Replace("\\", "/");
```

Comment:
Worth explaining in a comment that SonarQube/SonarCloud always uses / as the directory separator in patterns regardless of OS.